src/features/commands.ts

```diff
@@ -396,7 +390,7 @@ const commands: ChannelHandlers = {
       });
 
       if (keyword) {
-        if (cooldown.hasCooldown(msg.author.id, `commands.${keyword}`)) return;
+        // if (cooldown.hasCooldown(msg.author.id, `commands.${keyword}`)) return;
```

Comment:
Better to delete than comment, we have version control ```suggestion
```

Reply:
Oops this was a dev productivity change, didn't mean to commit